bool go2landing_config

---

bool success
